### DISCUSSION TITLE
Difference between Snowflake Git Integration and terraform-provider-snowflake

### DISCUSSION THREAD

**Discussion by aakashnand** [Snowflake-Labs/terraform-provider-snowflake#2682] [Q&A]
Hello Snowflake Community,

With the recent public preview of  [git connect to snowflake](https://medium.com/snowflake/connect-git-to-snowflake-now-in-public-preview-25b0456ce02c), I am confused what will be the correct usage for terraform provider and what will be the correct usage of git connect integration. 

I understand it will depend on each team and its own preference of managing DataOps but can someone from Snowflake or someone who is using both shed some light on how these two are different and what's their correct use case?

**Comment by sfc-gh-asawicki** [comment]
Hey @aakashnand. Thanks for reaching out to us.

The short answer would be that this is the alternative to using the terraform, and ultimately, it should be possible to achieve the same results using both approaches.

I will ask for more context from people leading the git integration initiative, and we will post it here.
